Multisig/UI/UI Library/LoadableView.swift

```diff
@@ -34,10 +35,25 @@ struct Loadable<V: LoadableView>: View {
             if model.isLoading {
                 ActivityIndicator(isAnimating: .constant(true), style: .large)
             } else if model.errorMessage != nil {
-                ErrorText(model.errorMessage!)
+                noDataView
             } else {
                 view
             }
         }
+        .onReceive(NotificationCenter.default.publisher(for: UIApplication.willEnterForegroundNotification)) { _ in
+            self.model.reloadData()
+        }
+    }
+
+    var noDataView: some View {
+        VStack {
+            HStack {
+                Image("ico-server-error")
+                Text("Data cannot be loaded").font(Font.gnoTitle3).foregroundColor(.gnoMediumGrey)
```

Comment:
Can you please use "TitleText" view instead?
Just update it to accept the color as init parameter like what we have in BodyText.swift

Reply:
done